src/main/java/com/github/wovnio/wovnjava/HtmlConverter.java

```diff
@@ -119,7 +107,13 @@ private void removeForm() {
         for (Element element : elements) {
             String type = element.attr("type");
             if (type != null && type.toLowerCase().equals("hidden")) {
-                replaceNodeToMarkerComment(element);
+                String original = element.outerHtml();
```

Comment:
This doesn't look correct 😉 Why is it removing attributes? The expected behavior is
```
<input type="hidden" value="some dynamic value">
->
<input type="hidden" value="__wovn__mark-3">
```

Reply:
Just values? I thought the idea is that it removes as much "dynamic" content as possible for better caching performance. Other than `values`, there are also stuff like `csrf_token` that needs to be removed. So I implemented this as removing all possible attributes.

Reply:
I would prefer if you just follow the implementation of the other backends for consistency

Reply:
Got it. Will change it to values only.